docker/resource_docker_container.go

```diff
@@ -238,7 +227,6 @@ func resourceDockerContainer() *schema.Resource {
 						},
 					},
 				},
-				Set: resourceDockerPortsHash,
```

Comment:
Very happy to see the custom `Set` methods go away 😄

Reply:
You're very welcome 🎆  🎉